include/defs.h

```diff
@@ -51,3 +51,12 @@
  */
 #define AOL_FILENAME "aol"
 
+/* xXx DEFINE=OL_SUCCESS xXx
+ * xXx DESCRIPTION=Used to indicate an operation went according to plan.
+ */
+#define OL_SUCCESS 0
```

Comment:
Bikeshedding: I like it more when TRUE means success and FALSE means error. Means you can do

```c
if (!do_some_shit()) {
    ... error handling here ...
}
```